lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -304,6 +521,83 @@ def app_name(self) -> str:
         return self.unit_name.split("/")[0]
 
 
+if int(pydantic.version.VERSION.split(".")[0]) < 2:  # type: ignore
```

Comment:
Are we aware of anyone that is pinned to old pydantic?

Reply:
not that I know of, no